src/views/activityBar/serverView/ServerDataProvider.ts

```diff
@@ -0,0 +1,50 @@
+// /src/views/server/ServerDataProvider.ts
+import * as vscode from "vscode";
+import { Shell } from "../../../utils/Shell";
+import { ServerStatus, ServerTreeItem } from "./ServerTreeItems";
+import { ServerStatusService } from "../../../services/ServiceStatusService";
```

Comment:
This import seems to be off as well

Reply:
Import should be fixed now. Thanks for the feedback @Cahllagerfeld.

And I 100% agree on having a github action for the tests. I'm setting up a workflow for automatic test execution on pushes and PRs, which will be included in the next PR.